src/hooks/useServerApi.ts

```diff
@@ -97,16 +98,26 @@ export const useServerApi = () => {
     authRequired = false,
     impersonationData,
     version,
-  }: Omit<RequestOptions, 'additionalHeaders'> & {
+    additionalHeaders,
```

Comment:
Allow `additionalHeaders` to be passed in all types of requests.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/2455/commits/6d0a99334e10374b77c13cb63f23f51e8892ea83